Tools/TestRunnerShared/UIScriptContext/Bindings/UIScriptController.idl

```diff
@@ -266,6 +266,7 @@ interface UIScriptController {
     readonly attribute boolean isDismissingMenu;
     readonly attribute boolean isShowingMenu;
     readonly attribute object menuRect;
```

Comment:
At some point we should rename this one to be clear it's about the callout bar.

Reply:
Yes! I _think_ we can go ahead and just rename it to something like `calloutBarRect` too, since it's only used on iOS anyways. I filed https://bugs.webkit.org/show_bug.cgi?id=241348 to track this.